CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- fixed a bug where comparing `ExtensionLocalValue` would fail is using `!=` operator [#304](https://github.com/Draegerwerk/sdc11073/issues/304)
```

Comment:
```suggestion
- fixed a bug where comparing `ExtensionLocalValue` would fail when using the `!=` operator [#304](https://github.com/Draegerwerk/sdc11073/issues/304)
```

Reply:
done